payNewUsers.js

```diff
@@ -50,10 +53,16 @@ async function getNewUsers(first) {
         }`);
         let users = res.allUsers.nodes;
         for (let i = 0; i<users.length; i++){
-            post(acornStack, `/pay @${users[i].id} ${amount} for joining Twetch!`, '', '', '');
-            console.log(`/pay @${users[i].id} ${amount} for joining Twetch!`);
+            let txid = post(wallet, `/pay @${users[i].id} ${amount} for joining Twetch!`, '', '', '');

```

Comment:
should we `await post` here? To wait for the post to complete before moving on in the loop? W/o i believe getNewUsers could complete before post is done

Reply:
yes we should. updating my branch with this now.